docs/conceptual-guides/choose-user-interface.md

```diff
@@ -0,0 +1,67 @@
+---
+myst:
+  html_meta:
+    "description": "Explanation of how to choose between Plone's two user interfaces, Volto and Classic UI"
+    "property=og:description": "Explanation of how to choose between Plone's two user interfaces, Volto and Classic UI"
+    "property=og:title": "Choose a user interface"
+    "keywords": "Plone 6, Conceptual guides, UI, user interface, frontend, Volto, Classic UI, distribution"
+---
+
+# Choose a user interface
+
+This guide explains the differences between Plone's two user interfaces, Volto and Classic UI, to help inform you to choose one when developing your new project in Plone, or whether to migrate from Classic UI to Volto.
+There is no migration path available from Volto to Classic UI.
+
+The choice of user interface has implications for editors, admins, and developers.
+
+`````{grid} 1 1 1 2
+:gutter: 1
+:margin: 0
+:padding: 0
+
+````{grid-item-card} Volto
+```{image} /_static/volto-ui.png
+:alt: Plone home page in Volto
+:target: /_static/volto-ui.png
+```
+
+For editors and other end users:
+
+-   The user interface is a fast, modern, single-page web application.
+-   Editors can create sophisticated page layouts by arranging {term}`blocks` of different types.
+-   There is not a comprehensive {doc}`/volto/user-manual/index` yet, and only a few pages exist.
+
+For developers and integrators:
+
+-   The frontend is a {term}`React`-based application written in JavaScript and TypeScript.
+-   The backend is a Python process which provides a {term}`REST API` for the frontend.
+-   Python skills are not required, but can be helpful for extending the backend.
+-   Content is stored as structured JSON.
+-   Customization of themes is well-documented and relatively easy for developers who have experience with React.
+
```

Comment:
Is this and its counterpart worth mentioning?

```suggestion
-   You cannot migrate to Classic UI.

```

Reply:
I was thinking of maybe adding a separate "Migrating between Volto and Classic UI" heading at the bottom of this page that covers this topic, but I think we're already at the point where it makes sense to merge this as is and make incremental improvements later.